clients/aws-sdk-transcribe-streaming/src/aws_sdk_transcribe_streaming/client.py

```diff
@@ -0,0 +1,378 @@
+    StartMedicalScribeStreamOutput,
+    StartMedicalStreamTranscriptionInput,
+    StartMedicalStreamTranscriptionOutput,
+    StartStreamTranscriptionInput,
+    StartStreamTranscriptionOutput,
+    TranscriptResultStream,
+    _CallAnalyticsTranscriptResultStreamDeserializer,
+    _MedicalScribeResultStreamDeserializer,
+    _MedicalTranscriptResultStreamDeserializer,
+    _TranscriptResultStreamDeserializer,
+)
+from .user_agent import aws_user_agent_plugin
+
+
+logger = logging.getLogger(__name__)
+
+
+class TranscribeStreamingClient:
+    """
+    Amazon Transcribe streaming offers four main types of real-time transcription: **Standard**, **Medical**, **Call Analytics**, and **Health Scribe**.
+
+    * **Standard transcriptions** are the most common option. Refer to for details.
+
+    * **Medical transcriptions** are tailored to medical professionals and incorporate medical terms. A common use case for this service is transcribing doctor-patient dialogue in real time, so doctors can focus on their patient instead of taking notes. Refer to for details.
+
+    * **Call Analytics transcriptions** are designed for use with call center audio on two different channels; if you're looking for insight into customer service calls, use this option. Refer to for details.
+
+    * **HealthScribe transcriptions** are designed to automatically create clinical notes from patient-clinician conversations using generative AI. Refer to [here] for details.
+
+    :param config: Optional configuration for the client. Here you can set things like the
+        endpoint for HTTP services or auth credentials.
+
+    :param plugins: A list of callables that modify the configuration dynamically. These
+        can be used to set defaults, for example.
+    """
+
+    def __init__(
+        self, config: Config | None = None, plugins: list[Plugin] | None = None
+    ):
+        self._config = config or Config()
+
+        client_plugins: list[Plugin] = [aws_user_agent_plugin, user_agent_plugin]
+        if plugins:
+            client_plugins.extend(plugins)
+
+        for plugin in client_plugins:
+            plugin(self._config)
+
+    async def get_medical_scribe_stream(
+        self, input: GetMedicalScribeStreamInput, plugins: list[Plugin] | None = None
+    ) -> GetMedicalScribeStreamOutput:
+        """
+        Provides details about the specified Amazon Web Services HealthScribe streaming
+        session. To view the status of the streaming session, check the ``StreamStatus``
+        field in the response. To get the details of post-stream analytics, including
+        its status, check the ``PostStreamAnalyticsResult`` field in the response.
+
+        :param input: The operation's input.
+
+        :param plugins: A list of callables that modify the configuration dynamically.
+            Changes made by these plugins only apply for the duration of the operation
+            execution and will not affect any other operation invocations.
+
```

Comment:
This looks like an issue in the docstring generator for operations? We're getting an unnecessary newline here but not in the top level class docstring.

Reply:
I'll check to see if these doc issue go away after https://github.com/smithy-lang/smithy-python/pull/592/files

Reply:
It does